proxy.py

```diff
@@ -28,8 +29,11 @@ async def netcat(port):
     # Verbose mode
     if os.environ["VERBOSE"] == "1":
         command.append("-v")
-    command += [f"{mode}-listen:{port},fork,reuseaddr,max-children={max_connections}",
-                f"{mode}-connect:{ip}:{port}"]
+    if mode.lower() == "udp" and upd_answers == "0":
+        command += [f"udp-recv:{port},reuseaddr", f"udp-sendto:{ip}:{port}"]
```

Comment:
Don't you need to add `udp-connect`, `udp-listen`, `fork` and `max-children` options in this mode? 🤔

Reply:
We use `udp-recv` and `udp-sendto` instead of `udp-listen` and `udp-connect`. They also forward the packets but do not require an answer.

Because they do not require an answer they do not fork (nor do they support the `fork` option) any subprocesses.

Because we therefore do not fork any subprocesses the `max-children` limit becomes obsolete for this variant.

Reply:
Thanks, great explanation.